### DISCUSSION TITLE
pr problem

### DISCUSSION THREAD

**Discussion by alireza-hdri** [community/community#63621] [Pull Requests]
### Select Topic Area

Bug

### Body

i've forked a repository and i have open some pull requests and all of them merged successfully.
from two or three days ago , when i open a pull request and it's merged the repository which i forked still shows that you are ahead from main repository.
this is the address of my [repo](https://github.com/alireza-hdri/ClusterBench) which is foreked and it shows i am about 40 commit ahead.
but i have opened pull request and they are merged.
there is no change in setting of both repositories expect there was a pipeline in the main repository with github action and it is deleted.
i think it can be a kind of bug.
i'll appreciate if you help me with this annoying issue.

**Comment by MayaAtwan** [comment]
Hey,

No worries, this sort of thing happens sometimes. Here's what you can do:

1) Check if there are any branches or commits still hanging around.
2) Make sure all the stuff you did in the pull request is actually done.
3) Sometimes GitHub takes a bit to catch up and show the latest. Just give it a moment.

Hope this helps :)
